examples/windows/WindowsExpCertSample/FlexAssets/get-expiringcerts.ps1

```diff
@@ -0,0 +1,14 @@
+$Certs = get-childitem -path cert: -recurse -Expiringindays 30 | Select-Object Issuer, NotAfter, Subject, FriendlyName
+$StartDate = (Get-Date)
+$CertsToSend = @()
+Foreach ($Cert in $Certs) {
+    $CertsObject = @{
+        ExpiringIn = New-TimeSpan -Start $StartDate -End $Cert.NotAfter
+        ExpirationDate = $Cert.NotAfter | Get-Date -Uformat %s
+        Issuer = $Cert.Issuer
+        Subject = $Cert.Subject
+        FriendlyName = $Cert.FriendlyName
+    }
+    $CertsToSend += $CertsObject
+
+}
```

Comment:
Does this script output any variable?

Reply:
Hey @rogercoll below is a sample output from the host and the table from newrelic. 

```
$CertsToSend

Name                           Value                                                                                                                                
----                           -----                                                                                                                                
Subject                        CN=Gondeh-tf-W-vm                                                                                                                    
FriendlyName                                                                                                                                                        
Issuer                         CN=Gondeh-tf-W-vm                                                                                                                    
ExpirationDate                 1684181133                                                                                                                           
ExpiringIn                     95.04:25:57.1673348   
```                                                                                                               


and here is a screenshot of the full data from my account:
![image](https://user-images.githubusercontent.com/50554236/217861708-caa6a94b-0b19-46c3-8767-cd42c5e1070b.png)


and lastly full select keyset() on the new table

```
[
  {
    "results": [
      {
        "stringKeys": [
          "FriendlyName",
          "Issuer",
          "Subject",
          "agentName",
          "agentVersion",
          "context_error",
          "coreCount",
          "displayName",
          "entityGuid",
          "entityId",
          "entityKey",
          "entityName",
          "environment",
          "error",
          "error_exec",
          "error_msg",
          "event_type",
          "fullHostname",
          "hostStatus",
          "hostname",
          "image",
          "instanceType",
          "integrationName",
          "integrationVersion",
          "integration_name",
          "integration_version",
          "name",
          "operatingSystem",
          "processorCount",
          "provider",
          "providerAccountId",
          "providerAccountName",
          "regionName",
          "resourceGroup",
          "resourceGroupName",
          "service",
          "subscriptionId",
          "systemMemoryBytes",
          "team",
          "type",
          "vmId",
          "vmSize",
          "windowsFamily",
          "windowsPlatform",
          "windowsVersion"
        ],
        "numericKeys": [
          "ExpirationDate",
          "ExpiringIn.Days",
          "ExpiringIn.Hours",
          "ExpiringIn.Milliseconds",
          "ExpiringIn.Minutes",
          "ExpiringIn.Seconds",
          "ExpiringIn.Ticks",
          "ExpiringIn.TotalDays",
          "ExpiringIn.TotalHours",
          "ExpiringIn.TotalMilliseconds",
          "ExpiringIn.TotalMinutes",
          "ExpiringIn.TotalSeconds",
          "criticalViolationCount",
          "timestamp",
          "warningViolationCount"
        ],
        "booleanKeys": [],
        "allKeys": [
          "ExpirationDate",
          "ExpiringIn.Days",
          "ExpiringIn.Hours",
          "ExpiringIn.Milliseconds",
          "ExpiringIn.Minutes",
          "ExpiringIn.Seconds",
          "ExpiringIn.Ticks",
          "ExpiringIn.TotalDays",
          "ExpiringIn.TotalHours",
          "ExpiringIn.TotalMilliseconds",
          "ExpiringIn.TotalMinutes",
          "ExpiringIn.TotalSeconds",
          "FriendlyName",
          "Issuer",
          "Subject",
          "agentName",
          "agentVersion",
          "context_error",
          "coreCount",
          "criticalViolationCount",
          "displayName",
          "entityGuid",
          "entityId",
          "entityKey",
          "entityName",
          "environment",
          "error",
          "error_exec",
          "error_msg",
          "event_type",
          "fullHostname",
          "hostStatus",
          "hostname",
          "image",
          "instanceType",
          "integrationName",
          "integrationVersion",
          "integration_name",
          "integration_version",
          "name",
          "operatingSystem",
          "processorCount",
          "provider",
          "providerAccountId",
          "providerAccountName",
          "regionName",
          "resourceGroup",
          "resourceGroupName",
          "service",
          "subscriptionId",
          "systemMemoryBytes",
          "team",
          "timestamp",
          "type",
          "vmId",
          "vmSize",
          "warningViolationCount",
          "windowsFamily",
          "windowsPlatform",
          "windowsVersion"
        ]
      }
    ],
    "metadata": {
      "contents": [
        {
          "function": "keyset"
        }
      ],
      "eventTypes": [
        "WindowsExpCertSample"
      ],
      "eventType": "WindowsExpCertSample",
      "openEnded": true,
      "messages": [],
      "beginTimeMillis": 1675955545867,
      "endTimeMillis": 1675957345867,
      "beginTime": "2023-02-09T15:12:25Z",
      "endTime": "2023-02-09T15:42:25Z",
      "guid": "790e57be-ab03-1576-de05-94d70d14f9b9",
      "routerGuid": "790e57be-ab03-1576-de05-94d70d14f9b9",
      "rawSince": "30 MINUTES AGO",
      "rawUntil": "NOW",
      "rawCompareWith": "",
      "accounts": [
        3291700
      ]
    },
    "performanceStats": {
      "inspectedCount": 2,
      "responseTime": 34
    }
  }
]
```

Reply:
Nice! Isn't the script missing printing the `$CertsToSend` variable at the end of the script?

Reply:
From my experience, printing `$CertsToSend` isn't required to send it. From what I have seen as long as the session ends with the json object we are good to go. I can add a printing line but I've read that isn't powershell best practice since its not a user session and a print to screen isn't necessary.

Reply:
wait, I may have pushed the wrong version in that case you're absolutely right, checking!

Reply:
you are beyond right! I couldn't fully get what you were asking! I'll fix it! the version I have running is one ahead of this push somehow. Thanks for pointing that out! @rogercoll